functions/db/import_histograms.fn.js

```diff
@@ -0,0 +1,95 @@
+const admin = require('firebase-admin');
+const { BigQuery } = require('@google-cloud/bigquery');
+const functions = require('firebase-functions');
+const { murmurHash128 } = require('murmurhash-native');
```

Comment:
If this project targets [a platform that supports `ECMAScript modules`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#Browser_compatibility), these could be rewritten to be more modern like so:

```javascript
import admin from 'firebase-admin';
import { BigQuery } from '@google-cloud/bigquery';
import functions from 'firebase-functions';
import { murmurHash128 } from 'murmurhash-native';
```

Reply:
Cloud functions are node.js and it looks like it would require naming these as `.mjs` files.

Reply:
Ah yeah. That may be true. I think newer versions of Node do away with that requirement but I'm not certain.